File: Volume 27 (2700 - 2799)/2786/2786.c
#include <stdio.h>
 
int main() {
    int largura, comprimento, perimetro, tipo1, tipo2;
    scanf("%d%d", &largura, &comprimento);
    perimetro = 2*largura + 2*comprimento;
    tipo1 = largura*comprimento + (largura-1)*(comprimento-1);
    tipo2 = perimetro - 4;
    printf("%d\n%d\n", tipo1, tipo2);
    return 0;
}

File: Volume 27 (2700 - 2799)/2786/2786.rb
#!/usr/bin/env ruby

largura = gets.to_i
comprimento = gets.to_i
perimetro = 2*largura + 2*comprimento
tipo1 = largura*comprimento + (largura-1)*(comprimento-1)
tipo2 = perimetro - 4
puts tipo1
puts tipo2